proto/reaction.proto

```diff
@@ -1100,18 +1100,20 @@ message Percentage {
   optional float precision = 2;
 }
 
-// Data is a container for arbitrary string or bytes data.
+// Data is a container for arbitrary data.
 message Data {
   oneof kind {
-    string string_value = 1;
-    float float_value = 2;
+    float float_value = 1;
+    int32 integer_value = 2;
```

Comment:
Do we have no persisted data yet?

Reply:
There's no persistent data in `ord-submissions-test` yet, if that's what you're asking. We'll blow away the stuff there and regenerate after these changes go in just to make sure everything is clean.